src/interface/cli/mod.rs

```diff
@@ -0,0 +1,78 @@
+use clap::{App, Arg, SubCommand, crate_authors, crate_version};
+use crate::readers;
+use crate::writers;
+use crate::writers::dir_builder::DirectoryBuilder;
+use std::path::Path;
+use std::process::Command;
+
+pub fn parse_args() {
+    let app_name = "apigen";
+    let about = "Generates directories and skeletons for APIs from .toml input file";
+    let matches = App::new(app_name)
+        .author(crate_authors!())
+        .version(crate_version!())
+        .about(about)
+        .subcommand(
+            SubCommand::with_name("build")
+                .about("Generate directory and associated files from .toml")
+                .arg(Arg::with_name("file")
+                    .help("The .toml file to read from")
+                    .required(true)
+                    .index(1)))
+        .subcommand(
+            SubCommand::with_name("run")
+                .about("Execute \"cargo run\" in target directory")
+                .arg(Arg::with_name("path")
+                    .help("The name of the folder in which to execute \"cargo run\"")
+                    .required(true)
+                    .index(1)))
+        .get_matches();

```

Comment:
This works but passing paths in as args to a CLI that _already_ has a user-written config file feels redundant.

The path should be able to be read from the toml file (maybe add a new method in the parser).

Reply:
so the toml file should be the arg?